src/kw_include.sh

```diff
@@ -24,6 +24,7 @@ function include()
   varname="${varname#"$KW_LIB_DIR/"}" # leave path until KW_LIB_DIR
   varname="${varname//\//_}"          # change bars to underlines
   varname="${varname%.*}"             # remove extension
+  varname="${varname/\./}"            # remove "." from hidden folder
```

Comment:
I think you can squash this commit with the previous one.

Reply:
I had separated them in an unusual way, let me see if I can make this any better.